python/perspective/perspective/libpsp.py

```diff
@@ -22,7 +22,13 @@ def is_libpsp():
     # dependent on `libpsppy` is exposed.
     from .table import *
     from .manager import *
-    from .viewer import *
+    # Only try importing viewer if traitlets is available
```

Comment:
less sure of this - traitlets is a none whl so it's not that bad to depend on it